runtime/gc_modron_startup/mmparseXXgc.cpp

```diff
@@ -243,26 +243,28 @@ gcParseXXgcArguments(J9JavaVM *vm, char *optArg)
 #endif /* J9VM_INTERP_NATIVE_SUPPORT */
 
 #if defined (OMR_GC_COMPRESSED_POINTERS)
-		/* see if we are to force disable shifting in compressed refs */
-		if (try_scan(&scan_start, "noShiftingCompression")) {
-			extensions->shouldAllowShiftingCompression = false;
-			continue;
-		}
-
-		/* see if we are forcing shifting to a specific value */
-		if (try_scan(&scan_start, "forcedShiftingCompressionAmount=")) {
-			extensions->shouldForceSpecifiedShiftingCompression = true;
-			if(!scan_udata_helper(vm, &scan_start, &(extensions->forcedShiftingCompressionAmount), "forcedShiftingCompressionAmount=")) {
-				returnValue = JNI_EINVAL;
-				break;
+		if (extensions->compressObjectReferences()) {
```

Comment:
I vote to remove this check. Setting this options is harmless for non-CR (but never used). I have two concerns:
 - possible race condition: do we know it would be CR or non-CR run at this point?
 - create internal dependancy between options in command line: line correct for CR run would fail if non-CR run would be selected - possibly automatically, for example based on max heap size

Reply:
This code runs in the ALL_LIBRARIES_LOADED stage - the compressed arguments have definitely been parsed by then, so the thread/vm will have the correct flags set, meaning environment and extensions will as well.

Reply:
I don't think we should be changing anything like this as part of this PR. My goal is to maintain identical function and performance in the normal builds for as long as possible.